app/code/Transoft/Blog/Ui/Component/Listing/Column/Actions.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Ui\Component\Listing\Column;
+
+use Magento\Framework\UrlInterface;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Framework\View\Element\UiComponentFactory;
+use Magento\Ui\Component\Listing\Columns\Column;
+
+/**
+ * Class ContactsActions
+ *
+ * UI component contacts actions class
+ */
+class Actions extends Column
+{
+    /**
+     * @var UrlInterface
+     */
+    protected $urlBuilder;
+
+    /**
+     * ContactsActions constructor.
+     *
+     * @param ContextInterface $context
+     * @param UiComponentFactory $uiComponentFactory
+     * @param UrlInterface $urlBuilder
+     * @param array $components
+     * @param array $data
+     */
+    public function __construct(
+        ContextInterface $context,
+        UiComponentFactory $uiComponentFactory,
+        UrlInterface $urlBuilder,
+        array $components = [],
+        array $data = []
+    ) {
+        $this->urlBuilder = $urlBuilder;
```

Comment:
Please use $this->context->getUrl. Context already has UrlInterface inside. 
See:
[Magento\Framework\View\Element\UiComponent\ContextInterface preference](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Ui/etc/di.xml#L14) and 
[Magento\Framework\View\Element\UiComponent\Context::getUrl](https://github.com/magento/magento2/blob/2.4-develop/lib/internal/Magento/Framework/View/Element/UiComponent/Context.php#L366)

Reply:
fixed